### ISSUE TITLE
Add a signifier to indicate which tab is active

### ISSUE THREAD

**Issue by neet** [neet/refined-itsukara-link#138]
Currently, the navigation bar tells users whether the tab is active by changing its text colour. 

However, since the colour contrast ratio is not enough, it is not clear for some colour blind people and lack in affordance

<img src="https://user-images.githubusercontent.com/19276905/82312952-b07fd080-9a02-11ea-8dc8-8a9b765a4aa4.png" alt="screenshot" width="200px" />
